IFC4x3/Constants/s/SOLID_0hCk8da6HBZAf0CKpzqv2y.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SOLID_0hCk8da6HBZAf0CKpzqv2y" Name="SOLID" UniqueId="2b32e227-9064-4b8c-aa40-314cfdd390bc">
+	<Documentation>Representation of the concept of an identified discrete almost homogenous solid geological or surface feature, including discontinuities such as faults, fractures, boundaries and interfaces that are not explicitly modelled.</Documentation>
+</DocConstant>
+
```

Comment:
IfcGeoScienceFeature from IfcTunnel will be covering this in another way.

Reply:
Agreed.